src/main/java/com/easypost/model/Shipment.java

```diff
@@ -1014,52 +1070,18 @@ public Rate lowestRate() throws EasyPostException {
      * @throws EasyPostException when the request fails.
      */
     public Rate lowestRate(final List<String> carriers) throws EasyPostException {
-        return this.lowestRate(carriers, null);
+        return this.lowestRate(null, null);
```

Comment:
This needs to have `carriers` passed in
```suggestion
        return this.lowestRate(carriers, null);
```